PWGMM/Lumi/Tasks/fitLumi.cxx

```diff
@@ -0,0 +1,116 @@
+
+struct fitLumi {
+  int nTF = 0;
+  HistogramRegistry registry;
+  int nBins, nCollBins;
+  int relTS;
+  Configurable<ULong64_t> startTimeInS{"startTime", 1668079200, "unix start time"};
+  Configurable<ULong64_t> endTimeInS{"endTime", 1668098700, "unix end time"};
+  Configurable<std::vector<int>> collBCArray{"collBCArray", {1022, 1062, 1102, 1142, 2015, 2055, 2161, 2201, 2241, 2281, 2321, 2361, 2401, 2441, 2481, 2521, 2561, 2601, 2641, 2681}, "Colliding BC for the VdM"};
+
+  void init(InitContext&)
+  {
+    nBins = ULong64_t(endTimeInS - startTimeInS) / 2.;
+    nCollBins = collBCArray->size();
+    // Hist for FT0
+    registry.add("FT0/VtxTrig", "vertex trigger;ts (s); Counts", {HistType::kTH1F, {{nBins, 0., static_cast<double>(endTimeInS - startTimeInS)}}});
+    registry.add("FT0/VtxTrigPerBC", "vertex trigger per BC;ts (s); Counts", {HistType::kTH2F, {{nBins, 0., static_cast<double>(endTimeInS - startTimeInS)}, {nCollBins - 1, 0., static_cast<double>(nCollBins)}}});
+    registry.add("FT0/TF", "TF ;ts (s); Counts (nTF)", {HistType::kTH1F, {{nBins, 0., static_cast<double>(endTimeInS - startTimeInS)}}});
+    registry.add("FT0/CollBCMap", "BC map for the colliding bcsr;BC entry; entries", {HistType::kTH1F, {{nCollBins - 1, 0., static_cast<double>(nCollBins)}}});
+    // Hist for FDD
+    registry.add("FDD/VtxTrig", "vertex trigger;ts (s); Counts", {HistType::kTH1F, {{nBins, 0., static_cast<double>(endTimeInS - startTimeInS)}}});
+    registry.add("FDD/VtxTrigPerBC", "vertex trigger per BC;ts (s); Counts", {HistType::kTH2F, {{nBins, 0., static_cast<double>(endTimeInS - startTimeInS)}, {nCollBins - 1, 0., static_cast<double>(nCollBins)}}});
+  }
+
+  using BCsWithTimestamps = soa::Join<aod::BCs, aod::Timestamps>;
+  void process(aod::FT0s const& ft0s, aod::FDDs const& fdds, aod::BCsWithTimestamps const&)
+  {
+    if (nTF < 1) {
+      for (int iBin = 0; iBin < nCollBins; iBin++) {
+        registry.get<TH1>(HIST("FT0/CollBCMap"))->SetBinContent(iBin + 1, collBCArray->at(iBin));
+        std::cout << "bin " << iBin << " value " << collBCArray->at(iBin) << std::endl;
+      }
+    }
+
+    for (auto const& ft0 : ft0s) {
+      auto bc = ft0.bc_as<BCsWithTimestamps>();
+      if (!bc.timestamp())
+        continue;
+      std::bitset<8> fT0Triggers = ft0.triggerMask();
+      [[maybe_unused]] auto localBC = bc.globalBC() % nBCsPerOrbit;
+      int index = FIND_INDEX(collBCArray, localBC);
+      bool vertex = fT0Triggers[o2::fdd::Triggers::bitVertex];
+      auto tsInSecond = ((bc.timestamp() * 1.e-3) - startTimeInS); // covert ts from ms to second
+      if (vertex) {
+        registry.get<TH1>(HIST("FT0/VtxTrig"))->Fill(tsInSecond);
+        if (index) {
+          registry.get<TH2>(HIST("FT0/VtxTrigPerBC"))->Fill(tsInSecond, index);
+        }
+      } // vertex
+    }   // ft0
+
+    for (auto const& fdd : fdds) {
+      auto bc = fdd.bc_as<BCsWithTimestamps>();
+      if (!bc.timestamp())
```

Comment:
The blocks after `if`, `for`, etc. should always be enclosed in braces `{}` as per ALICE code guidelines.

Reply:
done